packages/app-store/applecalendar/lib/CalendarService.ts

```diff
@@ -1,9 +1,9 @@
-import { Credential } from "@prisma/client";
+import { Prisma } from "@prisma/client";
 
 import CalendarService from "@calcom/lib/CalendarService";
 
 export default class AppleCalendarService extends CalendarService {
-  constructor(credential: Credential) {
+  constructor(credential: Prisma.CredentialCreateInput) {
```

Comment:
@zomars I believe this was the root of the type check errors.

Reply:
How? It was working well before.